CHANGES.md

```diff
@@ -1,10 +1,15 @@
 # json Changelog
 
-## json 9.0.5 (not yet released)
+## json 9.0.6 (not yet released)
 
 (nothing yet)
 
 
+## json 9.0.5
+
+- [issue #112] Fix `json -ga` performance degradation on long lines.
```

Comment:
Perhaps:

- [issue #112] Improve streaming (`json -ga`) performance for very long lines. For example, with a 1 MB JSON object on a line can be a 50x speed improvement. However, this is restricted to streaming of newline-separate JSON as opposed to adjacent JSON objects not separated by newlines (`{"a":1}{"a":2}`). The former case is expected to be much more common.